### ISSUE TITLE
macOS CI build now fails due to problem finding JPEG libraries...

### ISSUE THREAD

**Issue by edwardhartnett** [NOAA-EMC/NCEPLIBS-g2c#266]
Even with no changes to the code, the CI fails for MacOS when JPEG is used. The MacOS build is failing to find the JPEG library when building:

-- Could NOT find JPEG (missing: JPEG_LIBRARY JPEG_INCLUDE_DIR) 
CMake Error at /usr/local/Cellar/cmake/3.24.0/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Jasper (missing: JPEG_LIBRARIES) (found suitable version
  "2.0.25", minimum required is "2.0.25")

If I disable the Jasper cache, then the JPEG library is not found when jasper is built.

So there is some problem with the JPEG library install on MacOS.

We have:

```
    - name: install-dependencies
      run: |
        find /Library/Frameworks/ -name "png*"
        sudo rm -rf /Library/Frameworks/Mono.framework

        brew update
        brew install doxygen openjpeg 

```

And this used to work. Now it does not.

@DusanJovic-NOAA IIRC you set up the MacOS CI for g2c. Do you know what is happening here?

**Comment by DusanJovic-NOAA** [comment]
Maybe they changed the macOS VM and libjpeg is not installed by default anymore. Install it explicitly.
```
brew install doxygen openjpeg libjpeg
```

**Comment by edwardhartnett** [comment]
I couldn't get this working so I turned off Jasper builds for the macos. Now macos is only tested with openjpeg.

If this can be fixed it would be great, but I've got some other work under way and I can't wait to sort out this mac install problem. ;-(

**Comment by edwardhartnett** [comment]
This was fixed in #272
